.github/CODEOWNERS

```diff
@@ -6,20 +6,16 @@
 
 # Android/JVM
 
-jvm/ @sugarmanz @Brocollie08
```

Comment:
Tangential to the main purpose of the PR but I noticed in #95  I was tagged as a code owner because there were changes in the `core` directory under the jvm folder. These paths should be from the root, not relative.